games-action/powermanga/powermanga-0.93.1.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A 2D arcade shoot-em-up game"
+DESCRIPTION="Powermanga is an arcade 2D shoot-em-up game with 41 levels and more than 200 sprites.
+Join POWERMANGA, an elite unit tasked to clear space of pirates, and improve the safetly of peaceful voyagers and navigators.
+Collect gems to upgrade your ship (red, green), restore energy (purple) and gain satellite shields (yellow).
+Upgrades include mobility boost, guided missiles, rear fire shot, and more.
+Controls:
```

Comment:
Maybe an empty line above the "Controls".

It's a good idea to open the built package in HaikuDepot to see how the text looks.

Reply:
No need to add the control keys here

Reply:
@Begasus Are you sure? Why?